api/osrd_infra/views/stdcm.py

```diff
@@ -56,10 +61,10 @@ def make_stdcm_core_payload(request):
     res = {
         "infra": infra.pk,
         "expected_version": infra.version,
-        "rolling_stock": request["rolling_stock"].to_railjson(),
+        "rolling_stock": request["rolling_stock"].to_schema().dict(),

```

Comment:
I dont get the to_schema call, isn't it just dict()?

Reply:
The `request["rolling_stock"]` is a Django Model. The function `to_schema` will return a Pydantic model that can be serialize in a dictionary.

Reply:
gee ok I'm dumb thx